.github/workflows/tls.yml

```
@@ -30,6 +30,8 @@ jobs:
           --health-retries 10
     env:
       working-directory: ./scylla
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
```

```suggestion
```suggestion
        credentials:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
```